### PR TITLE
modernpro-coverletter:0.0.1

### PR DESCRIPTION
**Pull request by jxpeng98**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This is a cover letter template for job applications and other needs. The clean and modern sans fonts are easily customizable with `macfont` and `openfont`.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/modernpro-coverletter/0.0.1/modernpro-coverletter.typ [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      bottom-edge: "baseline",
+    )[#align(center, [#address])]
+    align(center)[#display(contacts)]
+    line(length: 100%, stroke: 0.5pt + primary_colour)
+  }, header-ascent: 1em)
+
+  // Add recipient details
+  recipientgenerate(
+    recipient.starttitle,
+    recipient.cltitle,
```
Comment (laurmaedje) [inline packages/preview/modernpro-coverletter/0.0.1/modernpro-coverletter.typ:186]: I get this error upon initialization:
```
error: dictionary does not contain key "cltitle"
    ┌─ @preview/modernpro-coverletter:0.0.1/modernpro-coverletter.typ:186:14
    │
186 │     recipient.cltitle,
    │               ^^^^^^^
```

File: packages/preview/modernpro-coverletter/0.0.1/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  name: "Exam Example",
+  address: [London, UK],
+  contacts: (
+    (text: "08856", link: ""),
+    (text: "example.com", link: "https://www.example.com"),
+    (text: "github.com", link: "https://www.github.com"),
+    (text: "123@example.com", link: "mailto:123@example.com"),
+  ),
+  recipient: (
+    starttitle: "Dear Committee Members,",
```
Comment (laurmaedje) [inline packages/preview/modernpro-coverletter/0.0.1/README.md:41]: kebab case, i.e. `start-title` would be more idiomatic here

File: packages/preview/modernpro-coverletter/0.0.1/modernpro-coverletter.typ (hunk 1 above)
Comment (jxpeng98) [reply packages/preview/modernpro-coverletter/0.0.1/modernpro-coverletter.typ]: Thanks! I fixed the error and tested the template from my side.

File: packages/preview/modernpro-coverletter/0.0.1/README.md (hunk 2 above)
Comment (jxpeng98) [reply packages/preview/modernpro-coverletter/0.0.1/README.md:41]: I change all double words variables into the kebab case, like `start-title`, `cl-title` and `font-type`.

**Pull request merged**